### PR TITLE
[1LP][RFR] Report sort column

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent

- __Adding tests__ 
    1. test_reports_sort_column
- __Enhancement__
    1. Enhance report's data view to implement sort column.
- __Fixing__
    1. Fix `Table` to sort data properly.

### PRT Run

{{ pytest: cfme/tests/intelligence/reports/test_reports.py -k "test_reports_sort_column"   -sqvvv }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/intelligence/reports/test_reports.py [hunk 1]
```diff
@@ -570,3 +570,44 @@ def test_reports_service_unavailable(temp_appliance_preconfig, file_name, restor
...
+            1. Go to Cloud Intel -> Reports -> All Reports
+            2. Select a report and queue it, make sure it's not empty.
+            3. Note the order of content of a targeted column.
+            4. Sort the targetted column on the basis of some property.
+            5. Traverse through all the rows, note the order of content of the targeted column.
+            6. Sort the first order and compare it with the second order.
+        expectedResults:
+            1.
+            2.
+            3. Both the orders must be same.
```
Comment (john-dupuy) [inline cfme/tests/intelligence/reports/test_reports.py]: Where are expected results 4-6?

Comment (valaparthvi) [reply cfme/tests/intelligence/reports/test_reports.py]: Yikes! I forgot those steps, will fix that!

**Comment by izapolsk** [comment]
LGTM. waiting for Prt

**Comment by valaparthvi** [comment]
@izapolsk I think this is G2G.

**Pull request merged**